### PR TITLE
Create conda-forge package for future to include script entry points

### PR DESCRIPTION
**Pull request by mmysinger**
Should fix https://github.com/ContinuumIO/anaconda-issues/issues/478

### CODE REVIEW

File: recipes/future/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: future
+  version: {{ version }}
+
+source:
+  fn: future-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/f/future/future-{{ version }}.tar.gz
+  sha256: 3d3b193f20ca62ba7d8782589922878820d0a023b885882deec830adbf639b97
+
+build:
+  number: 1
```
Comment (ocefpaf) [inline recipes/future/meta.yaml]: We usually start at `0`.

File: recipes/future/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  home: http://python-future.org
+  license: MIT
+  license_file: LICENSE.txt
+  summary: Clean single-source support for Python 3 and 2
+  description: |
+    Python-future  is the missing compatibility layer between Python 2 and
+    Python 3. It allows you to use a single, clean Python 3.x-compatible
+    codebase to support both Python 2 and Python 3 with minimal overhead.
+  doc_url: http://python-future.org/
+  dev_url: https://github.com/PythonCharmers/python-future
```
Comment (ocefpaf) [inline recipes/future/meta.yaml]: You need to add the `extra/maintainer` section here. See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L41-L45 for an example.

**Comment by mmysinger** [comment]
There is already a build 0 of 0.15.2 in the main conda recipes at ContinuumIO/anaconda-recipes/tree/master/future. Should I still use build 0?

File: recipes/future/meta.yaml (hunk 1 above)
Comment (mmysinger) [inline recipes/future/meta.yaml]: There is already a build 0 of 0.15.2 in the main conda recipes at ContinuumIO/anaconda-recipes/tree/master/future. Should I still use build 0?

**Comment by ocefpaf** [comment]
> There is already a build 0 of 0.15.2 in the main conda recipes at ContinuumIO/anaconda-recipes/tree/master/future. Should I still use build 0?

Yes. The build count is independent. Now `conda` has a channel preference we can just put `conda-forge` on top of `defaults` in the `.condarc` file and we will get this package instead of the existing one.

File: recipes/future/meta.yaml (hunk 2 above)
Comment (mmysinger) [inline recipes/future/meta.yaml]: Thanks, this is done and all automated checks are passing now

**Comment by jakirkham** [comment]
@edschofield, just letting you know we have added `future` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge future`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/future-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `future`. 😄

**Pull request merged**